src/adapters/postgres/user-adapter.ts

```diff
@@ -566,6 +668,22 @@ export class PostgresUserService {
     }
   }
 
+  public async validateFieldValues(field_value_array:string[]) {
```

Comment:
Why fieldValues code is added in this PR? @souravbhowmik1999

Reply:
This validation is required for duplicate field value data entry. We have already discussed that topic and implemented this validation on the cohort module